<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg_window</class>
 <widget class="QDialog" name="reg_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Регистрация</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_login">
      <property name="placeholderText">
       <string>Введите логин</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="placeholderText">
       <string>Введите пароль</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_firstName">
      <property name="placeholderText">
       <string>Введите имя</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_secondName">
      <property name="placeholderText">
       <string>Введите фамилию</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_polis">
      <property name="placeholderText">
       <string>Введите полис</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_bd">
      <property name="text">
       <string>Введите дату рождения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit_bd">
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2023</year>
        <month>1</month>
        <day>23</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>1920</year>
        <month>9</month>
        <day>14</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_price">
      <property name="text">
       <string>Введите цену услуги врача</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_price">
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_specialization">
      <item>
       <property name="text">
        <string>Выберите специальность врача</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_reg">
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
